vars/testKomodo.groovy

```diff
@@ -137,7 +137,6 @@ def call(Map args = [:]) {
                         source ${env.CI_SOURCE_ROOT}/${params.TEST_SCRIPT}
```

Comment:
IS the plan that the test script will run whatever it wants to run when sourced?